components/places/android/src/main/java/org/mozilla/places/Bookmarks.kt

```diff
@@ -0,0 +1,407 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.places
+
+import java.lang.RuntimeException
+
+
+/**
+ * Enumeration of the ids of the roots of the bookmarks tree.
+ *
+ * There are 5 "roots" in the bookmark tree. The actual root
+ * (which have no parent), and it's 4 children (which have the
```

Comment:
Micro-nit: "its".

Reply:
If we're doing micro-nits... s/have/has for "the actual root".